src/Security/Security/Cmdlets/Pricings/SetPricings.cs

```diff
@@ -43,16 +43,20 @@ public override void ExecuteCmdlet()
             var name = Name;
             var tier = PricingTier;
 
-            switch (ParameterSetName)
+
+            if(ParameterSetName == ParameterSetNames.InputObject)
             {
                 case ParameterSetNames.SubscriptionLevelResource:
```

Comment:
This is actually not in the Batch modules but it seems wrong to me? Since the switch case was removed the case statements should be removed too? I'm not sure this will compile as is?